test/e2e/network/dns.go

```diff
@@ -67,8 +68,9 @@ var _ = SIGDescribe("DNS", func() {
 	})
 
 	/*
-	   Testname: dns-for-services
-	   Description: Make sure that DNS can resolve the names of services.
+		Release : v1.9
+		Testname: DNS, services
+		Description: When a headless service is created, it MUST be able to resolve all the required service endpoints. A Pod running in the namespace MUST be able to resolve the service IP.
```

Comment:
"When a service is created, a pod in the same namespace must be able to resolve the service by all of the expected DNS names"

Reply:
Fixed.